tests/mock_wts_server.go

```diff
@@ -0,0 +1,98 @@
+package main
```

Comment:
package name also `tests`?

Reply:
nahh not for this one. a main package is required for the program to be executable. if it's package "tests", it won't build